Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: list test all clean
+
+BUILD = local/bin/awkward-kaitai-build
+JAVA_CLASSES = kaitai_struct_compiler/jvm/target/scala-2.12/classes/io/kaitai/struct
+
+all: src-animal/libanimal.so
+
+src-animal/libanimal.so: $(BUILD) src-animal/animal.cpp
+	PYTHONPATH=$$PYTHONPATH:local $(BUILD) src-animal/animal.cpp -b build
+
+$(BUILD):
+	pip install . -t local
+
+src-animal/animal.cpp: $(JAVA_CLASSES)
+	java -cp kaitai_struct_compiler/jvm/target/scala-2.12/kaitai-struct-compiler_2.12-0.11-SNAPSHOT.jar:/usr/share/kaitai-struct-compiler/lib/* io.kaitai.struct.JavaMain -t awkward --outdir src-animal example_data/schemas/animal.ksy
```

Comment:
this command points to the `/usr/share/kaitai-struct-compiler/lib/` folder which is standard in Ubuntu. Dont know of a way to do this that works on Windows/Mac as well.

Reply:
ok, tried this in [1ff5868](https://github.com/ManasviGoyal/kaitai_struct_awkward_runtime/pull/19/commits/1ff5868a039637ebbe63547e7d1ff2f441ccc45b), I don't have access to a system with WSL, so this is untested.

Reply:
I have WSL so I can test it for you. Are there any additional steps that I need to run?

Also, I think we should add the path for a Windows system (without WSL) too as we can't assume the user to have WSL.

Reply:
@ManasviGoyal it would be useful if you try to run `make test` and check if it works

can you test on Windows without WSL? I think it would be complicated to install all the dependancies under Windows without WSL.

Reply:
I am getting this error.
```
java -cp kaitai_struct_compiler/jvm/target/scala-2.12/kaitai-struct-compiler_2.12-0.11-SNAPSHOT.jar:wsl.localhost\Ubuntu\usr\share\kaitai-struct-compiler\lib\* io.kaitai.struct.JavaMain -t awkward --outdir src-animal example_data/schemas/animal.ksy
Error: Unable to initialize main class io.kaitai.struct.JavaMain
Caused by: java.lang.NoClassDefFoundError: scala/collection/immutable/List
make: *** [Makefile:19: src-animal/animal.cpp] Error 1
```

if i change this `wsl.localhost\Ubuntu\usr\share\kaitai-struct-compiler\lib\*` path to the relative path of my folder, it works.

Reply:
Also, please run `pre-commit run -a` and push the changes. Thanks.

Reply:
I ran pre-commit, it also wanted to reformat README and and endlines to some `ksy` files, but I didn't commit those changes.

I don't know how to fix the error under WSL and I cannot test it. What about merging this as it is now and then you created another PR with a fix for that path error?